civihr_employee_portal/templates/views-exposed-form-users_list.tpl.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * @file
+ * This template handles the layout of the views exposed filter form.
```

Comment:
you need to leave a comment here. Document why this file exists, and what are you changing of the original template